autogen/coding/docker_commandline_code_executor.py

```diff
@@ -164,36 +193,37 @@ def execute_code_blocks(self, code_blocks: List[CodeBlock]) -> CommandLineCodeRe
         files = []
         last_exit_code = 0
         for code_block in code_blocks:
-            lang = code_block.language
-            code = silence_pip(code_block.code, lang)
-
-            try:
-                # Check if there is a filename comment
-                filename = _get_file_name_from_content(code, Path("/workspace"))
-            except ValueError:
-                return CommandLineCodeResult(exit_code=1, output="Filename is not in the workspace")
+            lang = code_block.language.lower()
+            if lang not in self.SUPPORTED_LANGUAGES:
+                outputs.append(f"Unsupported language {lang}")
```

Comment:
For better log readability:
```suggestion
                outputs.append(f"Unsupported language {lang}\n")
```